marin/resources_utils.py

```diff
@@ -0,0 +1,14 @@
+
+
+
+device_flops_map = {
+    "V100": 112e12,
+    "A6000": 309.7e12 / 2,
+    "T4": 65e12,
+    "P4": 65e12,
+    "A10G": 125e12,
+    "A100": 312e12,
+    "H100": (1.513e15 / 2),
+
+    "TPU-V4": 275e12,
```

Comment:
can we read these off of https://github.com/stanford-crfm/levanter/blob/162ef5f321cfc2e14b1b9c1652e2cffd42b395e2/src/levanter/utils/flop_utils.py#L40 (the names will be different so you'll need to manually convert them but it would be good to keep them in sync

Reply:
need to add TPU-v5e, TPU-v6e, TPU-v5p

Reply:
Why don't we use the naming conventions in levanter then (maybe with "-" instead of " ")?